.github/workflows/auto-assign-owners.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Auto Assign'
+on:
+  pull_request_target:
```

Comment:
For reference: The `pull_request` trigger stated in the action's [docs](https://github.com/kentaro-m/auto-assign-action#arrow_forward-usage) will not work for PRs from forks since a GITHUB_TOKEN with write access to the target repository is required. See https://github.com/kentaro-m/auto-assign-action/issues/29.

Reply:
I know, I think I was the one found the solution :)

Reply:
Yep indeed, that was not directed to you but other curious readers 😄